packages/cozy-pouch-link/package.json

```diff
@@ -30,5 +30,8 @@
     "watch": "yarn run build --watch",
     "prepublishOnly": "yarn run build"
   },
+  "jest": {
+    "setupFiles": ["./jest.setup.js"]
```

Comment:
I am not finding this file in the repo, am I missing something ?

Reply:
Well spotted. This file is not used as jest is run once for the whole workspace instead of being run in each package individually. 

I put the fetch polyfill in the general setup file that is from cozy-client/packages/cozy-client. I'm moving this file to be at the root of the directory, it will be clearer.